test/JSON/jx2.js

```diff
@@ -187,7 +187,7 @@ function repf(key, value) {
 
 var objectArray = [
 { obj: "SampleTest", desc: "String simple" },
-{ obj: "/test ze\0ro\vString\n_u4:\u0061_u2:\xbc_u1:\x0e_u2clean:\x8f", desc: "String complex" },
+{ obj: "/test ze\0ro\vString\n_u4:\u0061_u2:\x62_u1:\x0e_u2clean:\x50", desc: "String complex" },

```

Comment:
Was this the only test input with non-ASCII characters (i.e., above `127`) in the project being passed to `JSON.stringify`? If so, this might reduce test coverage.